server/src/main/java/org/elasticsearch/cluster/metadata/IndexAbstractionResolver.java

```diff
@@ -29,32 +28,6 @@ public IndexAbstractionResolver(IndexNameExpressionResolver indexNameExpressionR
         this.indexNameExpressionResolver = indexNameExpressionResolver;
     }
 
-    public List<String> resolveIndexAbstractions(
-        String[] indices,
-        IndicesOptions indicesOptions,
-        Metadata metadata,
-        boolean includeDataStreams
-    ) {
-        return resolveIndexAbstractions(Arrays.asList(indices), indicesOptions, metadata, includeDataStreams);
-    }
-
-    public List<String> resolveIndexAbstractions(
-        Iterable<String> indices,
-        IndicesOptions indicesOptions,
-        Metadata metadata,
-        boolean includeDataStreams
-    ) {
-        final Set<String> availableIndexAbstractions = metadata.getIndicesLookup().keySet();
-        return resolveIndexAbstractions(
-            indices,
-            indicesOptions,
-            metadata,
-            () -> availableIndexAbstractions,
-            availableIndexAbstractions::contains,
-            includeDataStreams
-        );
-    }
-
```

Comment:
This whole class is going to be removed as it only adds a redundant layer on top of the `IndexNameExpressionResolver`.